src/MusicalScore/ScoreIO/MusicSheetReader.ts

```diff
@@ -486,21 +487,20 @@ export class MusicSheetReader /*implements IMusicSheetReader*/ {
     private pushSheetLabels(root: IXmlElement, filePath: string): void {
         this.readComposer(root);
         this.readTitle(root);
-        if (this.musicSheet.Title === undefined || this.musicSheet.Composer === undefined) {
-            this.readTitleAndComposerFromCredits(root);
-        }
-        if (this.musicSheet.Title === undefined) {
-            try {
+        try {
```

Comment:
Again this can't be the proper solution. I would check ```readTitleAndComposerFromCredits``` for the error first

Reply:
Why? You want two try catch blocks, just to know which try block the error came from?
If you want to know that, you can look at the error message or debug.
I don't really get what you mean, `readTitleAndComposerFromCredits` is the first thing i do in the try block anyways.

In any case, all of this works and gives proper error messages, so i don't see the problem.

Reply:
I understand now, fixed in 5dc8c80a3137db1e2e0abd4325ad1cd7260efd7f.
If `readTitleAndComposerFromCredits` fails, the second method of reading the title (from filename) wouldn't be executed with just one try block.